src/components/Section.js

```diff
@@ -5,8 +5,12 @@ export default class Section {
     this._container = document.querySelector(containerSelector);
   }
 
+  setItems(items) {
+    this._items = items;
+  }
+
   renderItems() {
-    this._items.forEach((item) => this._renderer(item));
+    this._items.slice(0, 6).forEach((item) => this._renderer(item));
```

Comment:
`.slice(0, 6)` is not needed here because you should render **all cards** of the user (not 6)